docs/en_US/NAS/classic_nas.rst

```diff
@@ -0,0 +1,11 @@
+Classic NAS Algorithms
```

Comment:
remove this file and put the content in RegularizedEvolutionTuner.md to ClassicNas.md